rust/src/utils.rs

```diff
@@ -223,7 +223,7 @@ pub fn maybe_shell_quote(input: &str) -> String {
 
 pub(crate) fn shellsafe_quote(input: Cow<str>) -> Cow<str> {
     lazy_static! {
-        static ref SHELLSAFE: Regex = Regex::new("^[[:alnum:]-._/=]+$").unwrap();
+        static ref SHELLSAFE: Regex = Regex::new("^[[:alnum:]-._/=:]+$").unwrap();
```

Comment:
This one could in theory be a prep commit with a new unit test, but fine as is!

Reply:
Done!